deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -159,6 +159,13 @@ def GenerateConfig(context):
 pip install -q --upgrade setuptools wheel
 pip install -q --upgrade -r requirements.txt
 
+# Setup Forseti logging
+touch /var/log/forseti.log
```

Comment:
Like above, you'll need to chown this to the user forseti server runs as.

Reply:
Done